molecule/builder/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # April 28th, 2018 ubuntu:trusty
```

Comment:
Hey can you update the comment from what image this hash is from?

Reply:
Thanks ! Fixed.